### DISCUSSION TITLE
[Support]: Are the annual model credits rolled over after subsequent years?

### DISCUSSION THREAD

**Discussion by ics460** [blakeblackshear/frigate#15420] [General Support] [answered]
With the 1-year anniversary of my (and probably many others) Frigate+ subscripting coming up, I'm curious what will happen with my remaining (5) credits and whether I should start using them up more frequently this next month.

Will credits rollover so I receive 12 additional credits on renewal, resulting in 17? Or will my total credits be set to 12?

**Comment by NickM-27** [comment]
I believe the credits do not roll over but @blakeblackshear can answer this question for sure

**Comment by blakeblackshear** [comment, marked as answer]
The 12 trainings with your annual subscription are meant to be used during the subscription. Your trainings count will reset to 12 when you renew, so they don't roll over. 

If your subscription is currently with Lemon Squeezy, canceling and resubscribing with my new payment processor within 30 days will reset to 17 trainings instead of the usual 12.

For those who run out and purchase additional trainings, those **will** rollover.

There isn't really a good reason to use them up just for the sake of it. The model doesn't improve from more frequent fine tuning. It improves with more or better training data.

**Reply by DaveTSG** [reply]
While we're on the topic, is there a way of buying additional trainings? I've only had Frigate+ for a few months and have almost run out of them as I keep getting (what I thought would be) rare edge-cases that need to be included in the training. I'm assuming it'll settle down soon...

**Reply by NickM-27** [reply]
Once you run out of credits the option will be there to buy more
